api/seeds/init.ts

```diff
@@ -0,0 +1,84 @@
+import bcrypt from 'bcryptjs'
+import { USER_PROFILE } from '@locokit/definitions'
+import { Knex } from 'knex'
+
+export async function seed(knex: Knex): Promise<any> {
+  const hashPassword = await bcrypt.hash('locokit', 10)
+
+  const trx = await knex.transaction()
+
+  /**
+   * User creation
+   */
+  const usersToInsert = [
+    {
+      firstname: 'Master',
+      lastname: 'ADMIN',
```

Comment:
In a previous mail, I suggested to modify (among other things)
```
firstname -> firstName
lastname -> lastName

```
to be consistent. 

This is just a reminder, maybe we can do it in this mr or maybe in the next.